packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add a new `DeFiPositionsController` that maintains an updated list of DeFi positions for EVM accounts ([#5400](https://github.com/MetaMask/core/pull/5400))
+  - Export `DeFiPositionsController`
+  - Export the following types
+    - `DeFiPositionsControllerState`
+    - `DeFiPositionsControllerAllowedActions`
+    - `DeFiPositionsControllerAllowedEvents`
```

Comment:
We don't typically export these types, can you remove these?

``` suggestion
```

Reply:
I will remove, but perhaps you want to review that restriction for the future, as with the new controller initialisation it just means that instead of just using this exported type, we have to import every individual type when creating the messenger.

Reply:
Gotcha. I will look into that.